sycl/test-e2e/Basic/sycl_2020_images/host_task_sampled_image_read_nearest.cpp

```diff
@@ -7,7 +7,7 @@
 
 #include "common.hpp"
 
-#include <sycl/detail/host_task_impl.hpp>
+#include <sycl/detail/core.hpp>
```

Comment:
Do we really need it here? I'd expect it to be part of `common.hpp`, same below.